### PR TITLE
ADEN-8517 Fix isSlotApplicable check

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: src/ad-bidders/prebid/prebid-helper.ts [hunk 1]
```diff
@@ -4,21 +4,32 @@ import { adaptersRegistry } from './adapters-registry';
...
-function isSlotApplicable(code, lazyLoad) {
-	const isSlotLazy = lazyLoadSlots.indexOf(code) !== -1;
+function isUsedAsAlias(code) {
+	let aliasFound = false;
 
-	if (!slotService.getState(code)) {
-		return false;
-	}
+	Object.keys(context.get('slots')).forEach((slotName) => {
+		const slot = context.get(`slots.${slotName}`);
```
Comment (fraszczakszymon) [inline src/ad-bidders/prebid/prebid-helper.ts]:
```suggestion
		const bidderAlias = context.get(`slots.${slotName}.bidderAlias`);
```

File: src/ad-bidders/prebid/prebid-helper.ts [hunk 2]
```diff
@@ -4,21 +4,32 @@ import { adaptersRegistry } from './adapters-registry';
...
+
+		if (slot.bidderAlias && slot.bidderAlias === code && slotService.getState(slotName)) {
```
Comment (fraszczakszymon) [inline src/ad-bidders/prebid/prebid-helper.ts]:
```suggestion
		if (bidderAlias === code && slotService.getState(slotName)) {
```

File: src/ad-bidders/prebid/prebid-helper.ts [hunk 3]
```diff
@@ -4,21 +4,32 @@ import { adaptersRegistry } from './adapters-registry';
...
 
-function isSlotApplicable(code, lazyLoad) {
-	const isSlotLazy = lazyLoadSlots.indexOf(code) !== -1;
+function isUsedAsAlias(code) {
+	let aliasFound = false;
 
-	if (!slotService.getState(code)) {
-		return false;
-	}
+	Object.keys(context.get('slots')).forEach((slotName) => {
```
Comment (fraszczakszymon) [inline src/ad-bidders/prebid/prebid-helper.ts]:
```suggestion
	return Object.keys(context.get('slots')).some((slotName) => {
```

File: src/ad-bidders/prebid/prebid-helper.ts [hunk 4]
```diff
@@ -4,21 +4,32 @@ import { adaptersRegistry } from './adapters-registry';
 const lazyLoadSlots = ['bottom_leaderboard'];
 const videoType = 'video';
 
-function isSlotApplicable(code, lazyLoad) {
-	const isSlotLazy = lazyLoadSlots.indexOf(code) !== -1;
+function isUsedAsAlias(code) {
+	let aliasFound = false;
```
Comment (fraszczakszymon) [inline src/ad-bidders/prebid/prebid-helper.ts]: Not needed

Comment (dmnsobczak) [reply src/ad-bidders/prebid/prebid-helper.ts]: Done!

File: src/ad-bidders/prebid/prebid-helper.ts (hunk 3 above)
Comment (dmnsobczak) [reply src/ad-bidders/prebid/prebid-helper.ts]: Done!

File: src/ad-bidders/prebid/prebid-helper.ts (hunk 1 above)
Comment (dmnsobczak) [reply src/ad-bidders/prebid/prebid-helper.ts]: Done!

File: src/ad-bidders/prebid/prebid-helper.ts (hunk 2 above)
Comment (dmnsobczak) [reply src/ad-bidders/prebid/prebid-helper.ts]: Done!

**Pull request merged**